dev/integration_tests/ui/lib/license_check.dart

```diff
@@ -14,18 +14,18 @@ void main() {
 }
 
 class ShowLicenses extends StatelessWidget {
-  const ShowLicenses({Key key}) : super(key: key);
+  const ShowLicenses({Key? key}) : super(key: key);
 
   Widget _buildTestResultWidget(
     BuildContext context,
     AsyncSnapshot<List<LicenseEntry>> snapshot,
   ) {
-    final List<LicenseEntry> entries = snapshot.data;
+    final List<LicenseEntry>? entries = snapshot.data;
```

Comment:
This should probably be this instead:
```suggestion
    final List<LicenseEntry> entries = snapshot.data ?? <LicenseEntry>[];
```